designs/EmbeddedReadmesTechSpec.md

```diff
@@ -0,0 +1,85 @@
+# Embedded Readmes Technical Spec
+
+* Status: Implementing
+* Author(s): [Advay Tandon](https://github.com/advay26)
+* Issue: [6873](https://github.com/NuGet/Home/issues/6873) README.md (markdown description) support for NuGet packages
+
+## Problem Background
+
+Users of NuGet want to be able to include Readme files as part of the package when they `pack` and `push` their project through the commandline. The [design document](https://github.com/NuGet/Home/wiki/Packaging-READMEs-within-the-nupkg) outlines the work needed for this feature and the intended results.
+
+## Who are the customers
+
+NuGet customers that use the commandline to publish their packages, and want to be able to include Readme files through the `pack` process, rather than having to do it manually after the package has been published.
+
+## Goals
+
+* Implement the Client side functionality for Embedded Readmes:
+    * nuget pack support
+    * dotnet pack support
+    * Validation of readme files
+
+## Non-Goals
+
+* Server side functionality
+* Parsing the readme as a valid markdown file on the Client side
+
+## Solution
+
+**NOTE:** To see the code described in this section, you can look at the edited files [here](https://github.com/NuGet/NuGet.Client/compare/dev-advay26-readme).
```

Comment:
Will the link be alive once this spec is merged? Right now it points to a new page.

Reply:
I've changed the link to point to my (Draft) PR instead of the branch I was working on so that the link will still be functional  even once the PR is merged and the branch is deleted.